datalad/support/gitrepo.py

```diff
@@ -1385,6 +1385,19 @@ def get_tracking_branch(self, branch=None):
 
         return track_remote, track_branch
 
+    @property
+    def count_objects(self):
+        """return dictionary with count, size(in KiB) information of git objects
+        """
+        try:
+            count_str, err = Runner().run(
+                ['git', 'count-objects', '-v'],
+                log_stderr=True, log_stdout=True)
+            count_list = count_str.replace(': ', '\n').split('\n')
+            count = {key: int(value) for (key, value) in zip(count_list[0::2], count_list[1::2]) if key and value}
+            return count
+        except CommandError:
+            raise(InvalidGitRepositoryError)
```

Comment:
I don't think we should try/except here and assume that it might fail only if repository is invalid... may be git experiences segfault or smth ;)  so why not to let CommandError to propagate and be done?